scripts/k8s-deployment-sim.sh

```diff
@@ -1,5 +1,20 @@
 #!/bin/bash
 
+printf "\n\n\n*** [0] install server with tls template and HA in k8s\n"
+yq e '.global.agtK8Config.withPlugins.tls.enabled = true' -i k8s/example/values.yaml
+yq e '.global.agtK8Config.withPlugins.vln.enabled = false' -i k8s/example/values.yaml
+helm install k8s/agent+lber --set-file global.agtConfig=k8s/example/server+tls.env --generate-name
+printf "\n\n\n*** [0.1] verify installation\n"
+kubectl get deployments && kubectl get pods && kubectl get services
+printf "\n\n\n*** [0.2] verify deployment spec\n"
+kubectl describe deployments $(kubectl get deployments|grep agent-plg|awk '{print $1}'|head -n 1)
```

Comment:
its should be a "lber" deployment instead.

Reply:
Updated to display deployment with name 'agentlber' and services 'agent' and 'agentlber'